internal/pkg/describe/service.go

```diff
@@ -182,7 +184,8 @@ type NewServiceConfig struct {
 	DeployStore     DeployedEnvServicesLister
 }
 
-func NewServiceDescriber(opt NewServiceConfig) (*ServiceDescriber, error) {
+// NewServiceStackDescriber instantiates the core elements of a new service.
+func NewServiceStackDescriber(opt NewServiceConfig) (*serviceStackDescriber, error) {
```

Comment:
nit: can this be private?
```suggestion
func newServiceStackDescriber(opt NewServiceConfig) (*serviceStackDescriber, error) {
```